packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -233,9 +297,13 @@ class TextTreeConfiguration {
   final String afterName;
 
   /// Text to add immediately after the description line of a node with
-  /// properties and/or children.
+  /// properties and/or children if he node has a body.
```

Comment:
"if he" => "if the"

Reply:
the joys of having a failing MacBook keyboard....